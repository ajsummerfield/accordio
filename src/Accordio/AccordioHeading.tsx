import React from 'react';

type AccordioHeadingProps = {
  children: any;
};

const AccordioHeading = ({ children }: AccordioHeadingProps) => {
  return (
    <div className="accordio__heading">
      {children}
      <div className="accordio__chevron"></div>
    </div>
  );
};

export default AccordioHeading;
