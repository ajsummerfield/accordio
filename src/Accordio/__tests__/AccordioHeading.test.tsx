import React from 'react';
import AccordioHeading from '../AccordioHeading';
import { shallowRenderer } from './helpers';

describe('AccordioHeading Component', () => {
  let component: JSX.Element;

  describe('when rendered', () => {
    beforeEach(() => {
      component = <AccordioHeading>Hello World</AccordioHeading>;
    });
  
    it('Should match the snapshot', () => {
      const tree = shallowRenderer(component);
      expect(tree).toMatchSnapshot();
    });
  });
});
