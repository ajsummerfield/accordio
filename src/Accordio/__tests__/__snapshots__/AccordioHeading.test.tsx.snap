// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccordioHeading Component when rendered Should match the snapshot 1`] = `
<div
  className="accordio__heading"
>
  Hello World
  <div
    className="accordio__chevron"
  />
</div>
`;
